Composer/cypress/integration/NotificationPage.spec.ts

```diff
@@ -9,6 +9,9 @@ context('Notification Page', () => {
   it('can show lg syntax error ', () => {
     cy.visitPage('Bot Responses');
 
+    // click the logo to clear any stray tooltips from page navigation
+    cy.findByAltText('Composer Logo').click();
```

Comment:
Do you think its worth moving this into the `visitPage` function?

Reply:
Done in another PR (based on the suggestion here).